### PR TITLE
[18.0][MIG] sale_partner_company_group: Migration to 18.0

### PR DESCRIPTION
**Pull request by nguyenminhchien**
Depend on 
- [x] https://github.com/OCA/partner-contact/pull/1871

oca-port: [blacklist PR(s) 1247 for sale_partner_company_group](https://github.com/OCA/partner-contact/pull/1874/commits/23ddf130c7c7aba43ed238c46f9829895d2f3627)

### CODE REVIEW

File: sale_partner_company_group/tests/test_sale_partner_company_group.py [hunk 1]
```diff
@@ -10,16 +10,6 @@ class TestSalePartnerCompanyGroup(TestBasePartnerCompanyGroup):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        # Remove this variable in v16 and put instead:
-        # from odoo.addons.base.tests.common import DISABLED_MAIL_CONTEXT
```
Comment (mmequignon) [inline sale_partner_company_group/tests/test_sale_partner_company_group.py]: Weren't we supposed to import DISABLED_MAIL_CONTEXT ?
Is it coming from `TestBasePartnerCompanyGroup->BaseCommon` ?

**Review by mmequignon — COMMENTED** [review]
Tiny remark, looking good

File: sale_partner_company_group/tests/test_sale_partner_company_group.py (hunk 1 above)
Comment (nguyenminhchien) [reply sale_partner_company_group/tests/test_sale_partner_company_group.py]: Yes, it is. https://github.com/OCA/partner-contact/blob/9e0fa464453ccad01c5b20a1b2f16613b3fedd96/base_partner_company_group/tests/test_base_partner_company_group.py#L7

**Review by sebalix — APPROVED** [review]

**Review by StefanRijnhart — APPROVED** [review]
Thanks!

**Pull request merged**